src/main/java/com/sedliarov/learningtable/repository/GroupRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.sedliarov.learningtable.repository;
+
+import com.sedliarov.learningtable.model.entity.Group;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+
+public interface GroupRepository extends JpaRepository<Group, Integer>{
+}
```

Comment:
And the same for other
```suggestion
public interface GroupRepository extends JpaRepository<Group, UUID>{
}
```